### DISCUSSION TITLE
Images upload but do not show.

### DISCUSSION THREAD

**Discussion by Xlaits** [mastodon/mastodon#19178] [Q&A]
Fresh mastodon instance, and uploads technically work, but they don't render at all.

Not sure where to start fixing this, honestly. Google has kind of let me down.

![image](https://user-images.githubusercontent.com/32484365/190312157-222dd757-e0e6-4d53-98e0-478ed63e9211.png)
![image](https://user-images.githubusercontent.com/32484365/190312198-f1f88376-a1bc-4220-865c-df5ff735e803.png)

Edit 1:
Apparently, the Avatars and Profile Banners refuse to load, as well.

![image](https://user-images.githubusercontent.com/32484365/190465538-7639e071-7ac9-4060-add1-4beb68811b82.png)

Edit 2:
Same with custom emoji...
![image](https://user-images.githubusercontent.com/32484365/190468942-7275cbcf-6f72-4b88-a2a1-a2800d43cae7.png)

**Comment by jazmichaelking** [comment]
Check your account's media settings in settings > preferences >appearance (yoursite.you/settings/preferences/appearance)

**Reply by Xlaits** [reply]
Why would that effect anything? It's not just me, it's every user.

**Comment by Xlaits** [comment]
I have added several updates here, but have not been seeing any other responses. I'm not sure what to do or what to add to help here.

**Reply by rcampoamor** [reply]
Seeing this exact same issue, but no answers.  Doesn't seem to matter if the image was locally uploaded for received from a remote stream - in both cases just the yellowish gradient image.  Clicking on the yellow image displays a popup with a broken image icon.
